en/getting_started/px4_basic_concepts.md

```diff
@@ -128,13 +128,20 @@ The companion computer will usually communicate using a MAVLink API like the MAV
 * [Robotics APIs](https://dev.px4.io/master/en/robotics/) (PX4 Developer Guide)
 
 
-## Removable Memory/Logging
+## SD Cards (Removable Memory) {#sd_cards}
 
-PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md) (SD support may not be present on every flight controller).
+PX4 uses SD memory cards for storing [flight logs](../getting_started/flight_reporting.md), and they are also required in order to use UAVCAN peripherals and fly [missions](../flying/missions.md).
+
+By default, if no SD card is present PX4 will play the [format failed (2-beep)](../getting_started/tunes.md#format-failed) tune twice during boot (and none of the above features will be available).
 
 > **Tip** The maximum supported SD card size on Pixhawk boards is 32GB.
+  The *SanDisk Extreme U3 32GB* is [highly recommended](https://dev.px4.io/master/en/log/logging.html#sd-cards) (Developer Guide).
 
-A number of recommended cards are listed in: [Developer Guide > Logging](http://dev.px4.io/en/log/logging.html#sd-cards)
+SD cards are never-the-less optional.
+Flight controllers that do not include an SD Card slot may:
```

Comment:
Added info about log streaming/missions here. Any good links to how this is done? If not, I guess we just merge.
This continues from https://github.com/PX4/px4_user_guide/pull/696/files#r397721035 https://github.com/PX4/px4_user_guide/pull/696/files#r397721035

Reply:
We have this: https://dev.px4.io/master/en/log/logging.html#log-streaming.
For missions: this is a board-config option: https://github.com/PX4/Firmware/blob/master/boards/intel/aerofc-v1/src/board_config.h#L115

Reply:
Thanks @bkueng . I added the logging link. The aero link is too low level for this doc so I added it as an HTML comment (for me future readers). Merging now.